BookPlayer/Services/ActionParserService.swift

```diff
@@ -149,6 +149,8 @@ class ActionParserService {
       SleepTimer.shared.setTimer(.off)
     case -2:
       SleepTimer.shared.setTimer(.endOfChapter)
+    case -3:
+      SleepTimer.shared.setTimer(.sticky)
```

Comment:
Re parsing a command for sticky, we can handle that in a future PR, to possibly setup an action to update user preferences